### ISSUE TITLE
Figma support

### ISSUE THREAD

**Issue by tomffee** [ant-design/ant-ux#18]
Hi I was just wandering, is there any roadmap about to release Figma file?

**Comment by hi-caicai** [comment]
I think about it ~

**Comment by billyv** [comment]
Figma supports Sketch files natively. So you can just load the Sketch file. It works fine. 

P.S. Thank you @hi-caicai for this nice template! Beautiful work.

**Comment by Prathik-Jain** [comment]
I was able to manage out something out of the sketch files :
[Figma - Ant Design 3.0](https://www.figma.com/file/XfgD8e1RxZOwuo8vAfLG7mdg/Ant.Design.3.0.Components?node-id=0%3A3990)

P.S : Most of the thing is in Chinese.
Anybody who want's to contribute to change things in English please comment below :tada:

**Comment by pixelwiz** [comment]
I found this, but it seems many components are missing, like data display (tables) https://www.figma.com/resources/assets/ant-design-ui-kit/

**Comment by mateusz-wierzbicki** [comment]
Hello everyone!
I just released Ant Design System for Figma! This is the largest UI kit for Figma too! I've been working on this nights and days! You can learn more about it here: www.antforfigma.com

**Comment by agustif** [comment]
> Hello everyone!
> I just released Ant Design System for Figma! This is the largest UI kit for Figma too! I've been working on this nights and days! You can learn more about it here: [www.antforfigma.com](http://www.antforfigma.com)

I literally imported the sketch files into figma yesterday as an OK solution, thank you so much for making this!!!

**Comment by mateusz-wierzbicki** [comment]
> > Hello everyone!
> > I just released Ant Design System for Figma! This is the largest UI kit for Figma too! I've been working on this nights and days! You can learn more about it here: [www.antforfigma.com](http://www.antforfigma.com)
> 
> I literally imported the sketch files into figma yesterday as an OK solution, thank you so much for making this!!!

You are welcome! It has almost all comonents along with their states. I also made sure to stick to ant design guidelines too! :)

**Comment by agustif** [comment]
> > > Hello everyone!
> > > I just released Ant Design System for Figma! This is the largest UI kit for Figma too! I've been working on this nights and days! You can learn more about it here: [www.antforfigma.com](http://www.antforfigma.com)
> > 
> > 
> > I literally imported the sketch files into figma yesterday as an OK solution, thank you so much for making this!!!
> 
> You are welcome! It has almost all comonents along with their states. I also made sure to stick to ant design guidelines too! :)

Would it be easy to apply my own theme to components, as using less/code? That's hard to review on the preview tbh.

**Comment by mateusz-wierzbicki** [comment]
> > > > Hello everyone!
> > > > I just released Ant Design System for Figma! This is the largest UI kit for Figma too! I've been working on this nights and days! You can learn more about it here: [www.antforfigma.com](http://www.antforfigma.com)
> > > 
> > > 
> > > I literally imported the sketch files into figma yesterday as an OK solution, thank you so much for making this!!!
> > 
> > 
> > You are welcome! It has almost all comonents along with their states. I also made sure to stick to ant design guidelines too! :)
> 
> Would it be easy to apply my own theme to components, as using less/code? That's hard to review on the preview tbh.

Yes but you need to edit styles in Figma to change colors and font. For example if your primary color is different from the one used in ant design, you will need to change styles from blue-1 to blue-10. You can learn more how to do it by reading [Getting Started](https://drive.google.com/file/d/14aWbe1Y2E9jknuNgEO47wgvzqf6mHguU/view).
